CharacterCreation/01_Races.txt

```diff
@@ -116,45 +153,73 @@ Starting Stat Points: 40
 Stat Modifiers: See Below
 Language: ECSC Standard
 
-Racial Traits:
-Pick any three augments (you may take a single augment no more than 2 times, 
+===== Racial Traits =====
+
+    Pick any three augments (you may take a single augment no more than 2 times, 

```

Comment:
Nonblocker: I don't like that the other two cyborg traits get Subsection headers but this one does not. It would be consistent to have a == Cybernetic Augmentation == subsection header between 156 and 158.